src/index.html

```diff
@@ -1,13 +1,53 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BOSE</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700;800&display=swap" rel="stylesheet"> -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+
+      <div class="container">
+
+        <div class="header__content">
+
+          <div class="header__nav">
+
+            <a href="#" class="logo">
```

Comment:
між батьківським і дочірніми тегами не треба відступи, тільки між сіблінгами
```suggestion
    <header class="header">
      <div class="container">
        <div class="header__content">
          <div class="header__nav">
            <a href="#" class="logo">
```